packages/cli/src/commands/tsgen.ts

```diff
@@ -0,0 +1,33 @@
+import type { CommandModule } from "yargs";
+import { loadStoreConfig } from "../config/loadStoreConfig.js";
+import { tsgen } from "../render-ts/tsgen.js";
+
+type Options = {
+  configPath: string;
+  out: string;
+};
+
+const commandModule: CommandModule<Options, Options> = {
+  command: "tsgen",
```

Comment:
I wonder if this should be `recsgen` or something since it might not be the only ts we generate

Reply:
that's actually why I called it tsgen - since I imagine it will generate enums and maybe other things aside from recs definitions